### PR TITLE
Status page

### PR DESCRIPTION
**Pull request by frankhuster**
`http://{server}/api/status` returns "OK" if no alerts were created or "n ALERTS" otherwise

### CODE REVIEW

File: api/src/main/java/org/motechproject/nms/api/web/StatusController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@Controller
+public class StatusController {
+    @Autowired
+    private AlertService alertService;
+
+    @RequestMapping("/status")
+    @ResponseBody
+    public String status() {
+        List<Alert> alerts = alertService.search(new AlertCriteria());
```
Comment (larubbio) [inline api/src/main/java/org/motechproject/nms/api/web/StatusController.java]: Will the blank AlertCriteria return all alerts?  Don't we want just those that are not Closed?

Comment (frankhuster) [inline api/src/main/java/org/motechproject/nms/api/web/StatusController.java]: Yeah, probably.

**Comment by koshalt** [comment]
@frankhuster can this PR be closed?

**Comment by frankhuster** [comment]
Yeah, nuke it.

**Comment by koshalt** [comment]
MY PLEASURE :grin: 
eod

**Comment by larubbio** [comment]
mythbusters-truck-explosion-o

**Comment by larubbio** [comment]
1945914_o

**Pull request closed without merging**